etc/ncp-config.d/nc-maintenance.cfg

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "nc-maintenance",
+  "name": "Nc-maintenanceMode",
+  "title": "nc-maintenance-mode",
+  "description": "Turn maintenance mode on or off",
+  "info": "Open maintenance mode (ACTIVE=yes), or close maintenance mode  (ACTIVE=no)",
```

Comment:
```suggestion
  "info": "Open maintenance mode (ACTIVE=yes), or close maintenance mode (ACTIVE=no)",
```

Reply:
:)

Reply:
Sorry, I'm not very likely to use this feature (suggested change), I've modified it. :)